uniflow/flow/__init__.py

```diff
@@ -1,9 +1,20 @@
 """Flow __init__ module."""
 # this register all possible flow into FlowFactory through
 # FlowFactory.register(cls.__name__, cls) in Flow __init_subclass__
-from uniflow.flow.flow import LinearFlow  # noqa: F401
-from uniflow.flow.model_flow import (  # noqa: F401;
-    HuggingFaceModelFlow,
-    JsonModelFlow,
-    OpenAIModelFlow,
+from uniflow.extract.flow.extract_txt_flow import ExtractTxtFlow  # noqa: F401, F403
+from uniflow.transform.flow.transform_huggingface_flow import (  # noqa: F401, F403
+    TransformHuggingFaceFlow,
 )
+from uniflow.transform.flow.transform_lmqg_flow import (  # noqa: F401, F403
+    TransformLMQGFlow,
+)
+from uniflow.transform.flow.transform_openai_flow import (  # noqa: F401, F403
+    TransformOpenAIFlow,
+)
+
+__all__ = [
+    "ExtractTxtFlow",
+    "TransformOpenAIFlow",
+    "TransformLMQGFlow",
+    "TransformHuggingFaceFlow",
+]
```

Comment:
nit: you should delete the flow folder and move everything up to the root uniflow folder to increase readability considering that there is only factory and base flow class here.